wildfly/unifiedpush-wildfly/README.md

```diff
@@ -18,45 +18,48 @@ Follow the [instructions](http://docs.docker.com/installation/)
 
 ## Running the image
 
-We need to run a Keycloak Server and a database for the UPS itself.
+Before starting the UPS we need to have a Keycloak Server and a database available for it.
 
-### Keycloak Server
-
-For the Keycloak instance run the following command:
-
-
-```shell
-docker run --name keycloakSRV \
-           -v /path/to/my/folder/containing/ups-realm:/keycloak-cfg \
-           -e KEYCLOAK_USER=admin \
-           -e KEYCLOAK_PASSWORD=admin \
-           jboss/keycloak:3.4.2.Final \
-           "-b 0.0.0.0 -Dkeycloak.import=/keycloak-cfg/ups-realm-sample.json"
-```
-
-For the database of the UnifiedPush Server itself, a similar command is needed:
+You can start a database for the UnifiedPush Server using the following command:
 
 ```shell
-docker run --name unifiedpushDB \
+$ docker run --name unifiedpushDB \
            -p 11306:3306 \
+           -p 8080:8080 \
+           -p 9090:9090 \
```

Comment:
why so many port mappings ?

Reply:
do we need the mgmt port ?

Reply:
@matzew we need to expose 8080 for Keycloak access and 9090 for UPS. I think the MySQL port can be removed, so I'll check that.